applications/adv_networking_bench/cpp/default_bench_op_rx.h

```diff
@@ -358,15 +372,17 @@ class AdvNetworkingBenchDefaultRxOp : public Operator {
   // TODO: make configurable?
   static constexpr int num_concurrent = 10;  // Number of concurrent batches processing
   // TODO: could infer with (batch_size / burst size)
-  static constexpr int MAX_ANO_BURSTS = 10;  // Batches from ANO for one app batch
+  static constexpr int MAX_BURSTS_PER_BATCH = 10;
 
   // Holds burst buffers that cannot be freed yet and CUDA event indicating when they can be freed
   struct BatchAggregationParams {
-    std::array<BurstParams*, MAX_ANO_BURSTS> bursts;
+    std::array<BurstParams*, MAX_BURSTS_PER_BATCH> bursts;
     int num_bursts;
     cudaEvent_t evt;
   };
 
+  int port_id_;                                    // Port ID to poll on
+  std::array<int, 16> queue_ids_;                  // Queue IDs to poll on
```

Comment:
if you want to stick to hardcoded for queue 0, let's just remove it. If we keep it, let's at least loop through all the non -1 queues?

Reply:
Switched this to loop over available queues